docs/tutorial/references.md

```diff
@@ -0,0 +1,73 @@
+---
+title: What's Next? I Need More!
```

Comment:
When we chatted last Friday, I think we discussed taking this "What's Next" page out of the document flow. Since it's a little bit redundant with the existing site navigation — especially once we update with the new nav.

That said, I'm fine with you leaving it in, as well! In that case, I would suggest just a little bit of prose to serve as connective tissue. I really miss the friendliness of the tutorial steps preceding. And to me, it's a value add. 

e.g.: 

Intro: Now you've tried Tilt with a sample app, here's an overview of what we offer in showing you how to set up Tilt on your own unique application.

Live Update: How to optimize your setup to get updates down from minutes to seconds.

Non-Kubernetes Resources: Tilt not only represents your services from Kubernetes, but can represent all the other parts of your dev env.

Extending Tilt: If you want to build your own Tilt client, or just make the Web UI work a bit differently...

Reply:
I took these suggestions so we wouldn't lose them AND hid the page 🤗